google-cloud-spanner/src/main/resources/com/google/cloud/spanner/connection/PG_ClientSideStatements.json

```diff
@@ -352,10 +355,53 @@
       "setStatement": {
         "propertyName": "SESSION\\s+CHARACTERISTICS\\s+AS\\s+TRANSACTION",
         "separator": "\\s+",
-        "allowedValues": "(READ\\s+ONLY|READ\\s+WRITE|ISOLATION\\s+LEVEL\\s+DEFAULT|ISOLATION\\s+LEVEL\\s+SERIALIZABLE)",
+        "allowedValues": "(((?:\\s*|\\s*,\\s*)READ\\s+ONLY|(?:\\s*|\\s*,\\s*)READ\\s+WRITE|(?:\\s*|\\s*,\\s*)ISOLATION\\s+LEVEL\\s+DEFAULT|(?:\\s*|\\s*,\\s*)ISOLATION\\s+LEVEL\\s+SERIALIZABLE)+)",
         "converterName": "ClientSideStatementValueConverters$PgTransactionModeConverter"
       }
     },
+    {
+      "name": "SET DEFAULT_TRANSACTION_ISOLATION =|TO 'SERIALIZABLE'|SERIALIZABLE",
+      "executorName": "ClientSideStatementSetExecutor",
+      "resultType": "NO_RESULT",
+      "statementType": "SET_READONLY",
+      "regex": "(?is)\\A\\s*set\\s+default_transaction_isolation(?:\\s*=\\s*|\\s+to\\s+)(.*)\\z",
+      "method": "statementSetPgDefaultTransactionIsolation",
+      "exampleStatements": [
+        "set default_transaction_isolation=serializable",
+        "set default_transaction_isolation to serializable",
+        "set default_transaction_isolation to 'serializable'",
+        "set default_transaction_isolation = 'serializable'",
+        "set default_transaction_isolation = \"SERIALIZABLE\""
+      ],
+      "setStatement": {
+        "propertyName": "default_transaction_isolation",
+        "separator": "(?:=|\\s+TO\\s+)",
+        "allowedValues": "(SERIALIZABLE|'SERIALIZABLE'|\"SERIALIZABLE\")",
+        "converterName": "ClientSideStatementValueConverters$PgTransactionIsolationConverter"
+      }
+    },
+    {
+      "name": "SET DEFAULT_TRANSACTION_READ_ONLY = TRUE|FALSE",
+      "executorName": "ClientSideStatementSetExecutor",
+      "resultType": "NO_RESULT",
+      "statementType": "SET_READONLY",
+      "regex": "(?is)\\A\\s*set\\s+default_transaction_read_only(?:\\s*=\\s*|\\s+to\\s+)(.*)\\z",
+      "method": "statementSetReadOnly",
+      "exampleStatements": [
+        "set default_transaction_read_only = true",
+        "set default_transaction_read_only = false",
+        "set default_transaction_read_only = t",
+        "set default_transaction_read_only = f",
+        "set default_transaction_read_only to 't'",
+        "set default_transaction_read_only to \"f\""
```

Comment:
We can add a couple of example statements here related to ON and OFF

Reply:
Done